Adding further troubleshooting advice in the tutorial (#340)
OfficeDev/office-scripts-docs: docs/tutorials/excel-tutorial.md

    |Limes |600 |500 |
    |Grapefruits |900 |700 |

> 3. Open the **Automate** tab. If you do not see the **Automate** tab, check the ribbon overflow by pressing the drop-down arrow. If it is still not there, follow the advice in the article [Troubleshoot Office Scripts](../testing/troubleshooting.md#automate-tab-not-appearing-or-office-scripts-unavailable).

Reviewer: Consider adding contractions. 
```suggestion
3. Open the **Automate** tab. If you don't see the **Automate** tab, check the ribbon overflow by pressing the drop-down arrow. If it's still not there, follow the advice in the article [Troubleshoot Office Scripts](../testing/troubleshooting.md#automate-tab-not-appearing-or-office-scripts-unavailable).
```
